src/Entity/Token.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Pucene\Bundle\DoctrineBundle\Entity;
+
+/**
+ * Token entity.
+ */
+class Token
```

Comment:
We need another entity - else we would duplicate a lot the tokens

Token .             TokenField                     Field
+ value      ->    + pos/start/end .     ->